README.md

```diff
@@ -1454,8 +1456,10 @@ Long-term capital gains taxes
 
 ### Federal taxes
 
-🄳 **Income tax** is the money paid by individuals to federal, state, and, in some cases,
-local governments, and includes taxation of ordinary income and capital gains.
+🄳 **Income tax** is the money paid by individuals (generally, U.S. citizens, residents, and
+some [foreigners](https://www.irs.gov/individuals/international-taxpayers/nra-withholding))
```

Comment:
@jlevy also wants to add this link for "[file](https://www.irs.gov/publications/p17#en_US_2017_publink1000170407) and pay"

Reply:
Move parenthetical to end sentence:

Generally, U.S. citizens, residents, and
some [foreigners](https://www.irs.gov/individuals/international-taxpayers/nra-withholding) [must file](https://www.irs.gov/publications/p17#en_US_2017_publink1000170407) and pay federal income tax.

Reply:
Fixed.